packages/diagrams-demo-gallery/demos/demo-animation/index.tsx

```diff
@@ -28,7 +27,7 @@ class NodeDelayedPosition extends React.Component<any, any> {
 
 export default () => {
 	//1) setup the diagram engine
-	var engine = createEngine();
+	var engine = createEngine({ repaintDebounce: true, repaintDebounceMs: 12 });
```

Comment:
can remove the boolean now :)

Reply:
@dylanvorster 
done! sorry, it was a typo

Reply:
@dylanvorster any news about this MR? 
It's kind of needed lately :)